src/VuetifyResource.vue

```diff
@@ -72,52 +72,63 @@
+                                color="green"
+                                dark
+                                fab
+                                slot="activator"
+                                small
+                                v-if="canUpdateResources(selected) && selected.length === 1"
+                                v-on:click="openUpdateHandler()"
+                                v-on="on"
+                            >
+                                <v-icon>$vuetify.icons.edit</v-icon>
+                            </v-btn>
+                        </template>
                         <span>{{ lang('update') }}</span>
                     </v-tooltip>
 
                     <v-tooltip left>
-                        <v-btn
-                            color="indigo"
-                            dark
-                            fab
-                            slot="activator"
-                            small
-                            v-if="canAdd === true"
-                            v-on:click="openCreateHandler()"
-                        >
-                            <v-icon>$vuetify.icons.add</v-icon>
-                        </v-btn>
+                        <template v-slot:activator="{on}">
+                            <v-btn
+                                color="indigo"
+                                dark
+                                fab
+                                slot="activator"
+                                small
+                                v-if="canAdd === true"
+                                v-on:click="openCreateHandler()"
+                                v-on="on"
+                            >
+                                <v-icon>$vuetify.icons.add</v-icon>
+                            </v-btn>
+                        </template>
                         <span>{{ lang('create') }}</span>
                     </v-tooltip>
 
                     <v-tooltip left>
-                        <v-btn
-                            color="red"
-                            dark
-                            fab
-                            slot="activator"
-                            small
-                            v-if="canDeleteResources(selected) && selected.length >= 1"
-                            v-on:click="showDeleteConfirmation()"
-                        >
-                            <v-icon>$vuetify.icons.delete</v-icon>
-                        </v-btn>
+                        <template v-slot:activator="{on}">
+                            <v-btn
+                                color="red"
+                                dark
+                                fab
+                                slot="activator"
+                                small
+                                v-if="canDeleteResources(selected) && selected.length >= 1"
+                                v-on:click="showDeleteConfirmation()"
+                                v-on="on"
```

Comment:
Does this work with multiple v-on's?

Reply:
It does work, and it's necessary for the `v-slot:activator`